src/main/java/com/github/tomakehurst/wiremock/extension/MappingsSourceExtension.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2023 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.extension;
+
+import com.github.tomakehurst.wiremock.standalone.MappingsSource;
+
+public abstract class MappingsSourceExtension implements MappingsSource, Extension {}
```

Comment:
Suggest we make this an interface rather than abstract class so that extensions wishing to implement more than one extension point in addition to this one can do so.

Reply:
changed to interface

Reply:
hi @tomakehurst,

the changes has been done, can you please review it now and merge